### PR TITLE
Fix markdown margin

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of change

Fixed the markdown margin. If we want to "intend" everything we could just add a margin to `article`

## Type of change

- Bug fix (a non-breaking change which fixes an issue)

## How the change has been tested

Tested locally

## Change checklist

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have tested the wiki locally and tested that all external links work

### CODE REVIEW

**Comment by Dr-Electron** [comment]
> The banner are now looking weird with the wider content area. Possible to center the images via css?

Or we can just add a margin to the whole `article` as I said. Not sure what's the better solution

File: src/css/custom.css [hunk 1]
```diff
@@ -130,7 +130,7 @@ article .card {
 }
 
 .markdown {
-  margin: 2rem 3rem;
+  margin: 2rem 0rem;
```
[the remark this answers is not in the document]
Comment (kilianhln) [reply src/css/custom.css:133]: To fix the wider content area we could set a max-width in the banner width. @JSto91

[the remark this answers is not in the document]
Comment (JSto91) [reply src/css/custom.css:133]: sure, max-width could work too. 👍

**Comment by jlvandenhout** [comment]
> > The banner are now looking weird with the wider content area. Possible to center the images via css?
> 
> Or we can just add a margin to the whole `article` as I said. Not sure what's the better solution

We can't add it to the article, because that would trigger Mark again. The navigation buttons at the bottom are not part of the article. We need to give a margin though, to the entire doc item, here is why:

edge

**Review by jlvandenhout — APPROVED** [review]

**Comment by JSto91** [comment]
can we add `max-width: 862px;` for now?

**Comment by jlvandenhout** [comment]
> can we add `max-width: 862px;` for now?

As an alternative, since banners currently are always the first image after the article title, what about selecting the first image after the title if it exists and make that 100% the width of the article? This has the benefit that we don't hardcode arbitrary max widths as magical numbers and we retain our responsive design. The only downside is we rely on the current structure of heaving `title -> banner -> rest of content`, but I'm happy with that as a temporary solution till we have a more robust option (aka configuring the banner in the front matter).

The added custom CSS rule would then be:

```
/* Make banners 100% width of the article if present */
.markdown > h1 + p > img {
  width: 100%;
}
```

**Comment by JSto91** [comment]
> > can we add `max-width: 862px;` for now?
> 
> As an alternative, since banners currently are always the first image after the article title, what about selecting the first image after the title if it exists and make that 100% the width of the article? This has the benefit that we don't hardcode arbitrary max widths as magical numbers and we retain our responsive design. The only downside is we rely on the current structure of having `title -> banner -> rest of content`, but I'm happy with that as a temporary solution till we have a more robust option (aka configuring the banner in the front matter).
> 
> The added custom CSS rule would then be:
> 
> ```
> /* Make banners 100% width of the article if present */
> .markdown > h1 + p > img {
>   width: 100%;
> }
> ```

keep in mind that this makes the banners blurry until they are "fixed" 😅
screen

what do you think about centering?
screen2

**Comment by jlvandenhout** [comment]
Fair point about the blurring. Max width looked better then centering, so I went with the width, to be removed when we fixed the banners.

**Review by JSto91 — APPROVED** [review]

**Pull request merged**